resources/tentative/newssite/news-nuxt/components/assets/FacebookIcon.vue

```diff
@@ -0,0 +1,10 @@
+<template>
```

Comment:
from a performance point of view, personally I would suggest moving these to pure svg images (e.g. in `~/public/assets/facebook-icon.svg`) and referencing them with `<img>` unless you need to make them reactive to props. It will mean that:

1. they can be cached by the browser
2. we don't need to include the content of these SVGs in the JS bundle

Reply:
For our testing, we are not (yet) taking load time in account.